microshift_release_notes/microshift-4-12-release-notes.adoc

```diff
@@ -125,4 +125,13 @@ Issued: 2023-03-07
 
 {product-title} release 4.12.6 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:1037[RHBA-2023:1037] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:1034[RHBA-2023:1034] advisory.
 
+For the `TopoLVM image`, see link:https://catalog.redhat.com/software/containers/lvms4/topolvm-rhel8/63972de3adcb55263891b983?container-tabs=dockerfile[lvms4/topolvm-rhel8]
+
+[id="microshift-4-12-7-dp"]
+=== RHBA-2023:1166 - {product-title} 4.12.7 bug fix update
+
+Issued: 2023-03-13
+
+{product-title} release 4.12.7 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:1166[RHBA-2023:1166] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:1163[RHBA-2023:1163] advisory.
+
 For the `TopoLVM image`, see link:https://catalog.redhat.com/software/containers/lvms4/topolvm-rhel8/63972de3adcb55263891b983?container-tabs=dockerfile[lvms4/topolvm-rhel8]
```

Comment:
Should this sentence end with a period?

Reply:
Good catch! Will fix on all entries. Thanks!